main.py

```diff
@@ -0,0 +1,46 @@
+from transformers import pipeline
+import streamlit as st
+
+
+def get_text():
+    return st.text_area(
+        label="Введите текст-маску на английском или французском языках (желаемые для подбора слова замените на *? ):",
+        value="Diplomacy is the *? of telling *? to go to *? in such a way that they *?."
+    )
+
+
+@st.cache
+def unmask(text):
+    split_text = text.split("*?")
+    result_text = split_text[0]
+    results = [""]
+    for i in range(1, len(split_text)):
```

Comment:
A more accurate solution is to use the regular expression: `re.match` and iterate through all matches.